OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyLogic.cs

```diff
@@ -193,17 +189,14 @@ void ConnectionStateChanged(OrderManager om, string password, NetworkConnection
 						orderManager.IssueOrder(Order.Command("map " + uid));
 						Game.Settings.Server.Map = uid;
 						Game.Settings.Save();
-						newMapUid = null;
-						oldMapUid = null;
-						lastUpdatedUid = null;
 					});
 
 					// Check for updated maps, if the user has edited a map we'll preselect it for them
 					modData.MapCache.UpdateMaps();
 
 					Ui.OpenWindow("MAPCHOOSER_PANEL", new WidgetArgs()
 					{
-						{ "initialMap", lastUpdatedUid ?? map.Uid },
+						{ "initialMap", modData.MapCache.LastUpdatedMap ?? map.Uid },
```

Comment:
Can `LastUpdatedMap` ever be `null`? It is set in `LoadMap` (which would crash before setting it to a null value), and that runs already during mod initialisation. 🤔

Reply:
Confirmed that on bleed the map selected in the lobby is also selected when opening the map chooser, but on this PR always the map loaded last when initialising the mod is selected. (Until you change a map while the game is running and that becomes the permanently selected map.)

Reply:
Yes, this is a regression I though of earlier today. I will fix it in the near future

Reply:
fixed it